src/main/java/com/google/collegeplanner/servlets/DatastoreServlet.java

```diff
@@ -87,18 +91,22 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
         // Successful. The default status code response is 200.
         return;
       }
-      addCourses(coursesArray, response);
+
+      try {
+        addCourses(coursesArray);
+      } catch (IllegalArgumentException e) {
+        continue;
+      }
     } while (page++ < PAGE_LIMIT);
   }
```

Comment:
i just noticed that we don't return...anything? No `application/json`, no response message. Something to fix up in a separate PR.

Reply:
I'll create an issue.

Reply:
+1